language-plutus-core/src/Language/PlutusCore/Evaluation/Machine/Ck.hs

```diff
@@ -91,19 +89,19 @@ substituteDb varFor new = go where
 -- > s ▷ con cn     ↦ s ◁ con cn
 -- > s ▷ error A    ↦ ◆
 (|>)
-    :: (GShow uni, GEq uni, DefaultUni <: uni, Closed uni, uni `Everywhere` ExMemoryUsage)
-    => Context uni -> Term TyName Name uni () -> CkM uni (Term TyName Name uni ())
-stack |> TyInst _ fun ty        = FrameTyInstArg ty      : stack |> fun
-stack |> Apply _ fun arg        = FrameApplyArg arg      : stack |> fun
-stack |> IWrap ann pat arg term = FrameIWrap ann pat arg : stack |> term
-stack |> Unwrap _ term          = FrameUnwrap            : stack |> term
-stack |> tyAbs@TyAbs{}          = stack <| tyAbs
-stack |> lamAbs@LamAbs{}        = stack <| lamAbs
-stack |> bi@Builtin{}           = stack <| bi
-stack |> constant@Constant{}    = stack <| constant
-_     |> err@Error{}            =
+    :: (GShow uni, GEq uni, DefaultUni <: uni)
+    => Context uni -> Plain Term uni -> CkM uni (Plain Term uni)
+stack |> TyInst _ fun ty      = FrameTyInstArg ty     : stack |> fun
+stack |> Apply _ fun arg      = FrameApplyArg arg     : stack |> fun
+stack |> IWrap _ pat arg term = FrameIWrap () pat arg : stack |> term
```

Comment:
Unrelated, but why *does* `FrameIWrap` have a pointless unit argument?

Reply:
Maybe to match up with the frames of the CEK machine, where that argument is `ExMemory` and not pointless. Not sure. I can remove it.